tensorflow_addons/seq2seq/beam_search_decoder.py

```diff
@@ -226,16 +214,16 @@ def _check_batch_beam(t, batch_size, beam_width):
         "TensorArray reordering during the beam search." %
         (t if context.executing_eagerly() else t.name))
     rank = t.shape.ndims
-    shape = array_ops.shape(t)
+    shape = tf.shape(t)
     if rank == 2:
-        condition = math_ops.equal(shape[1], batch_size * beam_width)
+        condition = tf.equal(shape[1], batch_size * beam_width)
     else:
-        condition = math_ops.logical_or(
-            math_ops.equal(shape[1], batch_size * beam_width),
-            math_ops.logical_and(
-                math_ops.equal(shape[1], batch_size),
-                math_ops.equal(shape[2], beam_width)))
-    return control_flow_ops.Assert(condition, [error_message])
+        condition = tf.logical_or(
+            tf.equal(shape[1], batch_size * beam_width),
+            tf.logical_and(
+                tf.equal(shape[1], batch_size), tf.equal(shape[2],
```

Comment:
probably wrap the second tf.equal into next line to make it more readable.

Reply:
done